src/sql/src/session/vars.rs

```diff
@@ -1154,21 +1154,21 @@ pub const ENABLE_CONSOLIDATE_AFTER_UNION_NEGATE: ServerVar<bool> = ServerVar {
 mod grpc_client {
     use super::*;
 
-    pub const CONNECT_TIMEOUT: ServerVar<Duration> = ServerVar {
+    pub const CONNECT_TIMEOUT: ServerVar<Option<Duration>> = ServerVar {
         name: UncasedStr::new("grpc_client_connect_timeout"),
-        value: &Duration::from_secs(5),
+        value: &Some(Duration::from_secs(5)),
         description: "Timeout to apply to initial gRPC client connection establishment.",
         internal: true,
     };
-    pub const HTTP2_KEEP_ALIVE_INTERVAL: ServerVar<Duration> = ServerVar {
+    pub const HTTP2_KEEP_ALIVE_INTERVAL: ServerVar<Option<Duration>> = ServerVar {
```

Comment:
@MaterializeInc/surfaces would love a double check that if `ServerVar<Duration>` has already been written down for an env, it's safe to make it `ServerVar<Option<Duration>>`

Reply:
This is only persisted to the stash if `ALTER SYSTEM SET grpc_client_http2_keep_alive_interval ..` has been executed (almost certainly it has not). If mz is started up where that was run, and then this change is applied, then the written down value will be converted to a `Some` on restart, which means this is safe to do!